frontend/src/pages/document.tsx

```diff
@@ -22,14 +54,29 @@ export function DocumentPage({ params }: RouteComponentProps<{ documentId: strin
           />
           <TopBarTitle>{data?.name}</TopBarTitle>
         </TopBarPart>
-
         <TopBarPart>
-          <WorkerStatus documentId={params.documentId} />
+          <IconButton
+            icon={TbFileExport}
```

Comment:
We're currently adding a new icon set with every second pull request. They all look sort-of the same, but also all have a different vibe. At some point we should decide on one or two icon sets to use IMHO (but not necessarily in this PR)

Reply:
Sounds good but I would postpone that